src/scheduler.js

```diff
@@ -162,8 +162,14 @@ class Scheduler extends events.EventEmitter {
       errJson = `(error formatting JSON: ${e})`;
     }
 
-    this.notify.email({
-      address: email,
+    let email = this.createEmail(hook, err, errJson);
+    this.notify.email(email);
+    this.notify.lastEmail = _.merge({}, email, {err, errJson});
```

Comment:
probably for ```err``` and ```errJson``` we don't have this option, so what should I do set them in production code or leave the testing for ```content``` of email ?